cluster-autoscaler/cloudprovider/aws/README.md

```diff
@@ -175,6 +175,21 @@ If you'd like to scale node groups from 0, an `autoscaling:DescribeLaunchConfigu
 }
 ```
 
+## AWS::AutoScaling::AutoScalingGroup MixedInstancesPolicy
```

Comment:
Could we use a different title? I think a template resource reference may not clear to users.